README.md

```diff
@@ -1 +1,30 @@
-# Postman
\ No newline at end of file
+# Postman
+
+Use postman to test the functionality of the APIs you are interested in.
+
+### Importing
+
+Precondition: the project is cloned to your local computer. Postman app is running.
+1. In Postman application click Import on the Environments tab.
+   ![Environment importing](docs/images/1.jpg?raw=true)
+2. Drag-and-drop .json files from ./environment to the Import window. Click the Import button.
+   ![Environment importing](docs/images/2.jpg?raw=true)
```

Comment:
The [Environment importing] must have a meaningul message in case when file is not found, e.g. 'Drag-n-drop json collections', 'Press import button', etc.

Reply:
omg, forgot to change all alt texts.
Thank you!